src/content/en/fundamentals/accessibility/03-semantics-navigation/semnav-two.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: shared/narrow
+title: "Using Headings Effectively"
+description: "The role of heading structure in accessibility"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 2
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+First, let's reiterate an ealier point: *DOM order matters*, not only for focus order but for screen reader order. As you experiment with screen readers like VoiceOver and ChromeVox Lite, you'll find the heading list follows the DOM order rather than the visual order. 
```

Comment:
change ChromeVox Lite to ChromeVox.

ChromeVox Lite is a special thing that Alice used in our Udacity course but it's not an actual screen reader.